boards/mulle/include/periph_conf.h

```diff
@@ -363,9 +363,9 @@ static const spi_conf_t spi_config[] = {
  * @{
  */
 #define RTT_NUMOF           (1U)
-#define RTT_IRQ_PRIO        CPU_DEFAULT_IRQ_PRIO
 #define RTT_IRQ             RTC_IRQn
-#define RTT_ISR             isr_rtc_alarm
+#define RTT_IRQ_PRIO        10
```

Comment:
Don't we have a macro for this level (or for levels in general)?

Reply:
There's a default IRQ prio Maceo which sets the priority to 1, but I copied this from the other kinetis boards to match. I figured any refactoring of the IRQ priorities should update all the boards in bulk.

Reply:
Ok, got it.